_content/articles/notes-on-the-udana_ireland.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Notes on the Udana"
+authors:
+  - ireland
+external_url: "https://journal.equinoxpub.com/BSR/article/view/15266/23244"
```

Comment:
Make this the `source_url` (as the pdf there is the entire volume).  Let's have no external_url but add `file_links: [exclusive_01/Notes%20on%20the%20Udana%20-%20John%20Ireland.pdf]`